src/utils.ts

```diff
@@ -74,6 +75,14 @@ export function isDefaultNotificationDisabled() : boolean | undefined {
 	return notificationSetting;
 }
 
+export function getOpenAtStartupSetting() : boolean {
+	const openAtStartupSetting : string | undefined = vscode.workspace.getConfiguration().get(DIDACT_OPEN_AT_STARTUP);
+	if (openAtStartupSetting && openAtStartupSetting === `true`) {
```

Comment:
in Che-theia, `openAtStartupSetting` seems to a boolean somehow. Is it the case in VSCode as well?

Reply:
In vscode it will return a string afaik.

Reply:
`vscode.workspace.getConfiguration().get<boolean>` is *supposed* to return a boolean but I only get true out of it whether it's false or not. So I don't think that's what we're looking for

Reply:
in VSCode:
![image](https://user-images.githubusercontent.com/650571/90403743-ae7cea80-e0a1-11ea-80c6-8e0f889876ef.png)

next lines always return `false` :/

Reply:
how can `typeof` used on a string return boolean? that sounds weird.

Reply:
> how can `typeof` used on a string return boolean? that sounds weird. Even when doing `console.log(typeof 'true');` it returns `string` for me.
Welcome to the Javascript world! :laughing: 
Types in functions/methods are not part of the Javascript language. `.d.ts` files are like interfaces that are used to help the developers but these ones doesn't exist at runtime.

Reply:
You are right. I just easily forget about that when coming from Java :D

Reply:
Would that be fine?
```ts
	const openAtStartupSetting : boolean | undefined = vscode.workspace.getConfiguration().get(DIDACT_OPEN_AT_STARTUP, false);
	return openAtStartupSetting;
```

Reply:
So actually we could simply go with `return vscode.workspace.getConfiguration().get(DIDACT_OPEN_AT_STARTUP, false);` and just need to also adapt the return types of the function?